ServerCore/Pages/Teams/Hints.cshtml

```diff
@@ -114,11 +114,15 @@ You have <b>
                         <form method="post" style="display:inline-block">
                             @if (Model.Team.HintCoinCount >= hint.AdjustedCost)
                             {
+                                var clickString = hint.AdjustedCost == 0
+                                    ? "return confirm('Are you sure you want to unlock this free hint?')"
```

Comment:
I like the direction, but: Do we even need to have confirmation for free hints? Seems unnecessary. Maybe "return true" or "".

Reply:
I think it'll be good to still have a confirmation for two reasons.
1. Solving pride. Sometimes I want to solve a puzzle with no hints even if they were free so I don't want to even accidentally see a hint.
2. It would be good to keep the same pattern of confirmation for hints so that people will always know that they will have less chance of "accidentally" unlocking  a hint

Reply:
Works for me! Thanks for the rationale.